pages/txtjs.html

```diff
@@ -4,10 +4,56 @@
         <meta charset="UTF-8">
         <meta name="viewport" content="width=device-width, initial-scale=1">
         <title>txt.js</title>
-        <link href="../styles/styles.css" rel="stylesheet">
+        <link href="../styles/stylesTodo.css" rel="stylesheet">
     </head>
     <body>
-        <h1>Acess granted!</h1>
-        <h2>Welcome to txtjs 🔥</h2>
+        <!-- <h1>Access granted!</h1> -->
+        <header>
+            <div class="info-container">
+                <h2>Welcome to todo-txt.js ✍🏻</h2>
+                <p>A place where you can set your tasks following the todo-txt schema and see them in a kanban board.</p>
+            </div>
+        </header>
+        <button class="button" onclick="appearById()">Add new task</button>
```

Comment:
according to the comments in the other files, this should be called like

```suggestion
        <button class="button" onclick="appearById('modalContainer')">Add new task</button>
```